udata_front/theme/gouvfr/templates/dataset/resource/card.html

```diff
@@ -3,7 +3,7 @@
 {% set available = resource.extras['check:status']
     and resource.extras['check:status'] >= 200 and resource.extras['check:status'] < 400 %}
 {% set unavailable = resource.extras['check:status'] and resource.extras['check:status'] >= 400 %}
-{% set last_update = resource.published if resource.published > resource.modified else resource.modified %}
+{% set last_update = resource.harvest.modified_at if (resource.harvest and resource.harvest.modified_at > resource.modified) else resource.modified %}
```

Comment:
`resource.harvest` may be defined and `resource.harvest.modified_at` be None. Thus leading to `TypeError: '>' not supported between instances of 'NoneType' and 'datetime.datetime'`